src/Compatibility/Microsoft.DotNet.PackageValidation/ApiCompatRunner.cs

```diff
@@ -87,6 +99,58 @@ public void RunApiCompat()
             _dict.Clear();
         }
 
+        private IAssemblySymbol GetAssemblySymbolFromStream(Stream assemblyStream, MetadataInformation assemblyInformation, out bool resolvedReferences)
+        {
+            resolvedReferences = false;
+            HashSet<string> directories = null;
+
+            // In order to enable reference support for baseline suppression we need a better way
+            // to resolve references for the baseline package. Let's not enable it for now.
+            bool shouldResolveReferences = !_isBaselineSuppression && _referenceDirectories != null &&
```

Comment:
how about the users manually providing the references for the baseline package ? should we add that support ?

Reply:
yeah, @ericstj mentioned adding that extensibility point as well. I agree and we should add it, let's just do it in a separate PR so it's easier to review.